Makefile

```diff
@@ -1,10 +1,12 @@
 CODE_COVERAGE_REPORT_FOLDER = $(PWD)/build/test-coverage
 CODE_COVERAGE_REPORT_NAME_UNIT = $(CODE_COVERAGE_REPORT_FOLDER)/coverage-unit-report
+VER_PKG = github.com/elastic/elastic-package/internal/version
```

Comment:
Let's rename it to the:`VERSION_IMPORT_PATH`

Reply:
The choice was intentional to reduce line length. It's not far from its use, so the context is still strong. If you would still like me to make that change given this, let me know.

Reply:
Yes, I suggest extending it as VER_PKG might be misleading.

Reply:
It's not clear to me how, but done.

Reply:
My first guess was PKG_CONFIG as it is a Makefile.

Thanks for adjusting it.